content/en/docs/languages/ruby/basics.md

```diff
@@ -334,6 +334,31 @@ resps.each do |r|
 end
 ```
 
+Streaming RPCs block until the server closes the connection. For non-blocking
```

Comment:
> Streaming RPCs block until the server closes the connection.

More accurate description here is that streaming RPCs block until the RPC is terminated (which can happen for multiple reasons - for example, the server sends all messages and a status, the client cancels the RPCs, the connection breaks, etc.). That said, this is the case for all RPC types (e.g. unary too as well as streaming). So I think we shouldn't include this text under here since it may be misleading.